llconf/nodes.c

```diff
@@ -377,7 +377,8 @@ char *cnfnode_path(struct cnfnode *cn)
 	*q0 = 0;
 
 	for(cn_parent = cn; cn_parent; cn_parent = cn_parent->parent){
-		char *p, *q;
+		const char *p;
+        char *q;
```

Comment:
misaligned

Reply:
I changed tabs to spaces in this file. Hope that's okay.